.github/quarkus-github-bot.yml

```diff
@@ -387,7 +387,7 @@ triage:
         - independent-projects/bootstrap/
         - independent-projects/tools/
     - labels: [area/documentation]
-      notify: [ebullient, quarkusio/Documentation]
+      notify: [ebullient, michelle-purcell, hmanwani-rh, sheilamjones, MichalMaler]
```

Comment:
@ebullient Would you please also add Sunayna ([sunayna15](https://github.com/sunayna15)) and me ([inoxx03](https://github.com/inoxx03)) to the notify list? For general awareness, triage, and writer backup in case some of the writers are on PTO when they get notified. Thank you:)

Reply:
HA. Somehow I forgot you?! :sigh: Where are my brains ..